src/SignalR/clients/cpp/test/signalrclienttests/internal_hub_proxy_tests.cpp

```diff
@@ -65,37 +65,6 @@ TEST(on, cannot_register_handler_if_connection_not_in_disconnected_state)
     }
 }
 
-//TEST(invoke_event, invoke_event_invokes_event_and_passes_arguments)
```

Comment:
Why are we removing this test and the one below?

Reply:
invoke_event was some internal hub_proxy thing for invoking the `.on` handler of a hub_connection. We don't have hub_proxy.

Reply:
I see. You should probably rename this class to something other than internal_hub_proxy_tests.cpp. I'd also clean up signalrclient/CMakeLists.txt as it still includes hub_proxy.cpp and internal_hub_proxy.cpp.

Reply:
Also signalrclientdll.vcxproj.